adapters/roulax/roulax.go

```diff
@@ -0,0 +1,144 @@
+package roulax
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+
+type ExtImpRoulax struct {
+	Pid   string `json:"Pid,omitempty"`
+	PublisherPath string `json:"publisherPath,omitempty"`
+}
```

Comment:
This declaration should be moved to openrtb_ext package in `imp_roulax.go` file.